components/src/List/List.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import {
+  space, color, fontSize, fontWeight, lineHeight,
+} from "styled-system";
+import theme from "../theme";
+import ListItem from "./ListItem";
+
+const List = styled.ul`
+  list-style: none;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Curious about `-webkit` style attributes in general: Should we be adding the attributes that target the other browsers as well?

Reply:
The firefox fontsmoothing is `-moz-osx-font-smoothing: grayscale;`

I wonder if we should be applying this globally though... it's not really a decision unique to List, rather we want all of our fonts to be antialiased. 

We could add it to `ThemeProvider.js` and have it come along when we set the font to Plex.

Reply:
I agree we should have this applied globally.

Reply:
Yeah. There's probably a library that provides these.
They usually come in sets of 3, right?

Reply:
Put this on our ThemeProvider. Ideally we can use a library to handle these attributes on different browsers so I made a new ticket to handle that.